Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -1,723 +1,693 @@
-//// Automatically Generated From Tokens.swift.gyb.
```

Comment:
Right now it also adds the file to the `gyb_generated` folder. 
I think it's fine to continue with that.

Reply:
Yeah, I think that’s fine for now.

Reply:
It did just hit me, that the degyb will remove all files, from the folder if there is not related gyb file. 

So made a new folder `generated`.